plugins/ua_config_standard.h

```diff
@@ -12,6 +12,7 @@ extern "C" {
 #include "ua_server_config.h"
 #include "ua_client.h"
 #include "ua_client_highlevel.h"
+#include "ua_types.h"
```

Comment:
Is this needed? Try to move it into .c file where it is needed or the specific header file